src/main/java/org/zendesk/client/v2/model/Page.java

```diff
@@ -0,0 +1,41 @@
+package org.zendesk.client.v2.model;
+
+import java.io.Serializable;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public abstract class Page<T extends SearchResultEntity> implements Serializable {
```

Comment:
Would probably add a javadoc of some sort to indicate that this is what allows us to support paging with zendesk